packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -24,7 +24,7 @@ function cellContent(isTitle, item, column, titleProps, id) {
 		/>);
 	}
 	return (
-		<TooltipTrigger label={item[column.key]} tooltipPlacement="top">
+		<TooltipTrigger label={`${item[column.key]}`} tooltipPlacement="top">
```

Comment:
add test showing that even if `item[column.key]` return undefined, label will be an empty string